core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -1837,6 +1858,7 @@ public void onSkipToNext() {
             UiModeManager uiModeManager = (UiModeManager) getApplicationContext()
                     .getSystemService(Context.UI_MODE_SERVICE);
             if (UserPreferences.getHardwareForwardButton() == KeyEvent.KEYCODE_MEDIA_NEXT
+                    || Build.VERSION.SDK_INT >= Build.VERSION_CODES.S
```

Comment:
Doesn't this make the "Forward button skips" setting unusable?

Reply:
Yes, I think it would. Looking at the code now, I think it would be better to just call mediaPlayer.skip() when the custom skip action is called, since we always want to skip in that case.